tooling/miniutil/src/build/expr.rs

```diff
@@ -135,6 +135,22 @@ pub fn local(x: u32) -> PlaceExpr {
     PlaceExpr::Local(LocalName(Name::from_internal(x)))
 }
 
+pub fn global<T: TypeConv>(x: u32) -> PlaceExpr {
+    let relocation = Relocation { 
+        name: GlobalName(Name::from_internal(x as _)), 
```

Comment:
Why `as _`? Just above we have a `from_internal` from a `u32` without `as`.

`as` is a red flag as data might get lost on a cast, so if it's not needed please remove it.

Reply:
There's a clippy lint for this, perhaps you should consider enabling it

Reply:
I copied this from `miniutil/src/build/function.rs` where it is used multiple times.
The only possible reason I could find for this is in libspecr where Name is defined and the doc comment says, that the `u32` is hidden on purpose.

Reply:
> I copied this from miniutil/src/build/function.rs where it is used multiple times.

Yes, over there `x` has type `usize` so a cast is needed. Here `x` already has the right type so it is not needed.